texascale-org/static/texascale-org/css/src/_imports/objects/o-offset-content.css

```diff
@@ -1,5 +1,18 @@
+/* Retain Texascale 2019–2021 styles */
 /* SEE: TACC/Core-CMS _imports/objects/o-offset-content.css */
 
-[class*="o-offset-content--"] {
+:--o-offset-content {
 	--offset-distance: 8.5vw;
 }
+
+@media only screen and (--medium-and-above) {
+  .figure:is(:--o-offset-content) .figure-caption {
+    padding-right: unset;
+  }
+}
+@media only screen and (--medium-and-above) and (--max-wide-and-below) {
+  .o-offset-content--left,
+  .o-offset-content--right {
+    max-width: 50%;
+  }
+}
```

Comment:
**Explanation.** Update styles based on [TACC/Core-Styles#16:`/source/_imports/objects/o-offset-content.css`](https://github.com/TACC/Core-Styles/pull/16/files#diff-4568efa6476eb9b665d81688a5b81e8d3b825de2d6503f47db5be6f749c0253b) changes.

<details>

- Use new customs elector `:--o-offset-content`.
- Retain Texascale's`--offset-distance` value.
- Do **not** use Core's newly-added `padding-right` value.
- Do **not** use Core's updated `max-width` value.

</details>